mqtt/mqtt-bridge/src/persist/waking_state/waking_map.rs

```diff
@@ -0,0 +1,57 @@
+#![allow(dead_code)] // TODO remove when ready
```

Comment:
mqtt_bridge::persist::waking_state::waking_map seems a bit too much of waking. lets rename
`waking_map` -> `memory`
`waking_store` -> `rocksdb`
?

Reply:
AFAIU Bridge will work with `Persistor<S>`, right? Let's rename `Persistor` to `Store` or `PublicationStore` or something else? Persistor has a bit more low-level meaning for me at least